debian/changelog

```diff
@@ -1,4 +1,9 @@
 daos (2.7.100-5) unstable; urgency=medium
+  [ Tomasz Gromadzki ]
+  * pmemobj errors and warnings reported via DAOS logging system
+
+ -- Tomasz Gromadzki <tomasz.gromadzki@intel.com>  Tue, 03 Sep 2024 12:00:00 +0200
+
```

Comment:
You have inserted your new comment into an existing entry here.  You need to move lines 2-6 above line 1 and add a new line above that with a new version-release.

Reply:
Fixed